Breaking: 88755 - Remove POST option from typolink.addQueryString (#390)
TYPO3-Documentation/TYPO3CMS-Reference-Typoscript: Documentation/Functions/Typolink.rst

   not check for any duplicate parameters! This is not a problem: Only
   the last parameter of the same name will be applied.

   .. attention::

      .. versionchanged:: 10.0

         With version 10.0 a breaking change was introduced. Setting
         :typoscript:`addQueryString.method` to any value but :typoscript:`GET` will

>          trigger a `E_USER_WARNING`. Any POST-parameters will be ignored. Formerly this option

Reviewer: ```suggestion
         trigger an `E_USER_WARNING`. Any POST-parameters will be ignored. Formerly this option
```
not sure.

         was set to :typoscript:`POST` or some combination in order to forward
         POST-parameters as GET parameters.

>          In terms of correctly using HTTP verbs it’s bad practise in
>          general to treat GET and POST equally, besides that documentation
>          already mentioned potential side-effects like accidentally exposing
>          sensitive data submitted via POST to proxies or log files.

Reviewer: ```suggestion
         In terms of correctly using HTTP verbs, it is bad practice in
         general to treat GET and POST equally. In addition, there can be
         potential side-effects like accidentally exposing sensitive data 
         submitted via POST to proxies or log files.
```
I wasn't sure about including the documentation if it's unclear where that was. Perhaps you could link to where it already mentioned it.

         That’s why values :typoscript:`POST`, :typoscript:`GET,POST` and
         :typoscript:`POST,GET` are not allowed anymore
         for typolink.addQueryString.method. Maintaining functionality - if
         required at all - has to be done using domain specific logic in
         according controllers or middleware implementations.

         Setting :typoscript:`addQueryString.method = GET` will forward GET

>          parameters and have the same functionality like not setting it at all.

Reviewer: ```suggestion
         parameters and have the same functionality as if not setting it at all.
```
